src/key.js

```diff
@@ -1,11 +1,11 @@
 /* @flow */
 'use strict'
 
-const path = require('path')
 const uuid = require('uuid/v4')
 
-const pathSepS = path.sep
-const pathSep = new Buffer(pathSepS, 'utf8')[0]
+const pathSepS = '/'
+const pathSepB = new Buffer(pathSepS, 'utf8')
```

Comment:
needs to change to `Buffer.from`

Reply:
see the following comment

Reply:
What I'm referencing here is the linting issue https://travis-ci.org/ipfs/interface-datastore/jobs/296689734#L2153